### ISSUE TITLE
[PT] Update lesson: Erlang Term Storage (ETS)

### ISSUE THREAD

**Issue by thiamsantos** [elixirschool/elixirschool#2401]
## Motivation

The portuguese translation for the lesson [Erlang Term Storage (ETS)](https://elixirschool.com/en/lessons/specifics/ets/) needs to be updated.
We need to update the version of the lesson from `1.1.0` to `1.1.1`.

## Proposed Solution

Translate the updates of the lesson.

You can run `git diff` to see all differences between the versions of the lesson:

```sh
git diff 04d27245806677343c585d0ca9217921dd3c3d44 HEAD en/lessons/specifics/ets.md
```

## Tips

- Check the [contributing guidelines](https://github.com/elixirschool/elixirschool/blob/master/CONTRIBUTING.md) before starting.
- Feel free to open a [draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/#tag-your-work-in-progress) as soon as you start working on the translation. In order to prevent we prevent having multiple people working on the translation and also allow the reviewers to give early feedback.
- In case of any doubt feel free to comment in this issue.

**Comment by herminiotorres** [comment]
I take this update lesson and I just sent a pull-request #2408

**Comment by thiamsantos** [comment]
Closed by #2408 

Thanks @herminiotorres :tada: :metal:
